spec/fabricators/user_fabricator.rb

```diff
@@ -6,4 +6,5 @@
   phone '8675309'
   image 'bologna'
   reputation 0
+  roles { [Fabricate.build(:role, name: "registered_user")] }
```

Comment:
I like that you assigned the role here. You saved us a bunch of lines in our tests!!!